packages/backend/src/handlers/actions/quests/createQuest/permissions.ts

```diff
@@ -12,8 +12,9 @@ const { BN, amountToDecimal } = numbers;
 export async function isAllowedToCreateQuest(
   playerAddress: string,
 ): Promise<boolean> {
+  console.log(CONFIG.pSEEDAddress);
```

Comment:
Left a bit of cruft.  FYI, at the bottom of our README I recently added instructions for debugging backend code using VS code. It has sped up my workflow quite a bit